environments/template/secrets/skeleton.yml

```diff
@@ -21,7 +21,7 @@ mongo_passwords:
 mongo_admin_password: secret 
 
 engine_janus_secret: secret
```

Comment:
This one is valid?

Reply:
`engine_janus_secret` seems to be missing on a few other places too

Reply:
I've left this out of scope even tho it is used in the `.ini` file. In my opinion, the complete removal of Janus, including the install and uninstall tasks should be outside of the scope of this PR.